### DISCUSSION TITLE
FAILED - RETRYING: Wait until the etcd cluster is healthy

### DISCUSSION THREAD

**Discussion by LeGrim94** [autobase-tech/autobase#131] [Q&A]
I keep getting this error (I opened the ports on my aws ec2-machine and modified the whole project with the rights IPs: previous steps are workin' fine. I get stuck here "name: Wait for port 2379 to become open on the host"). What can I do?

To be more specific

TASK [etcd : Wait until the etcd cluster is healthy] ***************************
fatal: [x.x.x.x]: FAILED! => {"attempts": 10, "changed": false, "cmd": ["/usr/local/bin/etcdctl", "cluster-health"], "delta": "0:00:00.009443", "end": "2021-11-30 16:39:33.397496", "msg": "non-zero return code", "rc": 5, "start": "2021-11-30 16:39:33.388053", "stderr": "", "stderr_lines": [], "stdout": "failed to check the health of member cdbb13876426f862 on http://x.x.x.x:2379: Get http://x.x.xx:2379/health: dial tcp x.x.x.x:2379: connect: connection refused\nmember cdbb13876426f862 is unreachable: [http://x.x.x.x:2379] are all unreachable\ncluster is unavailable", "stdout_lines": ["failed to check the health of member cdbb13876426f862 on http://x.x.x.x:2379: Get http://x.x.x.x:2379/health: dial tcp x.x.x.x:2379: connect: connection refused", "member cdbb13876426f862 is unreachable: [http://x.x.x.x:2379] are all unreachable", "cluster is unavailable"]}

**Comment by vitabaks** [comment]
Hi @LeGrim94 

You have an error at the stage of performing a cluster health check using the command `etcdctl cluster-health`

> connection refused\n member cdbb13876426f862 is unreachable

this indicates possible network problems, port 2379 (for member cdbb13876426f862) is not available.

it will also be useful to look at the etcd log on all nodes of the cluster:
`sudo journalctl -u etcd -n 100 -f --output=short-precise`
